.github/workflows/tests_cmake_valgrind.yml

```diff
@@ -42,7 +42,7 @@ on:
 
 jobs:  
   valgrind_check:
-    timeout-minutes: 60
+    timeout-minutes: 180
```

Comment:
Please lower this time as close as possible to the actual runtime of the valgrind action + 10min.

Reply:
Yes, done. Valgrind check needs ~1h 25 min